### PR TITLE
Add KeyStore Password Unmarshaler for file&value

### LINKED ISSUE #28: Implement a configuration type for private values
Implement a configuration type that has the following properties:
- It's an alias to either string or []byte
- The String() method doesn't show the contents
- It has a TextUnmarshall implementation that behaves like this:
  - If the marshalled type is of the form "file://..." it opens the file, reads the contents and stores it in the type
  - If the marshalled type is of the form "password://..." it removes the "password://" prefix and stores the rest in the type

### PR DESCRIPTION
**Pull request by arnaucube**
resolve #28

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -38,9 +39,50 @@ type Server struct {
...
+type Password struct {
+	Value string  // private content
+	Path  *string // path of the file with the password
+}
+
+func (p *Password) String() string {
+	if p.Path == nil {
+		return fmt.Sprintf("password://***")
+	}
+	return fmt.Sprintf("file:://%v", *p.Path)
```
Comment (ed255) [inline config/config.go]: typo in `file:://%v`

Comment (ed255) [reply config/config.go]: Actualy you can use the prefixes here.

File: config/config.go [hunk 2]
```diff
@@ -38,9 +39,50 @@ type Server struct {
...
+}
+
+const (
+	prefixPassword = "password://"
+	prefixFile     = "file://"
+)
+
+func (p *Password) UnmarshalText(data []byte) error {
+	// Password can be prefixed by two options
+	//   file: <path to file containing the password>
```
Comment (ed255) [inline config/config.go]: This needs to be updated to `://`

File: config/config.go (hunk 1 above)
Comment (arnaucube) [reply config/config.go]: Done

File: config/config.go (hunk 2 above)
Comment (arnaucube) [reply config/config.go]: Done

**Pull request merged**